sesman/chansrv/config.c

```diff
@@ -0,0 +1 @@
+#include "../config.c"
```

Comment:
That's strange, what is the idea here ? usually only headers are included. Here you duplicate all the file to be recompiled. For your feature, I thing you don't have to create file but just modify the  existing one.